network_wrangler/Scenario.py

```diff
@@ -38,14 +38,29 @@ def create_scenario(base_scenario: dict, card_directory: str = '', tags: [str] =
         base_scenario: the base scenario
         tags: only project cards with these tags will be read/validated
         folder: the folder location where the project cards will be
+        project_cards_names: project names
         '''
-        scenario = Scenario(base_scenario, project_cards = project_card_list)
 
-        if card_directory:
-            scenario.add_project_cards(card_directory, tags = tags)
+        project_cards = []
 
-        return scenario
+        prereq_dict   = OrderedDict()
+        coreq_dict    = OrderedDict()
+        conflict_dict = OrderedDict()
+
+        for project_name in project_card_names:
```

Comment:
We don't want to have to specify all the card names for all the projects that we might want to apply. We should either be able to:
1. specify a directory and a tag, whereby you end up iterating thru the directory in search of the tag
2. specify a list of project card objects [or dir+names, if you convince me it is better]
3. both (additive)

This function makes it so you are only looking at the named project cards.

Reply:
yeah, I saw that you had a function to bulk apply projects cards (with a tag) from a directory.

Co-requisite and Conflicts are not a problem but prerequisite is (based on my understanding so far). For prerequisite, order of how projects cards are applied matters. 

If the projects are read from a directory (looping over .yml files), how do we know in what order they are applied to the scenario? Prerequisite needs the order, correct?

Reply:
The reason I am carrying the list of project card **names** so that in the event of conflicts or missing requisite, you can report which project card (name) has the conflict project and such. If we just have the project card object, we can only report that there is a conflict issue and user will then need to look into the list of all project cards that are applied, to find the issues.

We could have a project card object attribute with `self.name` to simplify things perhaps?

Reply:
I think we need to separate out the queuing, ordering/checking pre-requisites, and then applying as separate steps. Make sense?

Reply:
Ooo, good point on the reporting.  If we have a project card object, it should the name within it though...right?